app/assets/javascripts/discourse/app/templates/user.hbs

```diff
@@ -431,81 +431,22 @@
       </div>
     </section>
 
-    {{#if this.currentUser.redesigned_user_page_nav_enabled}}
-      <div class="new-user-wrapper">
-        <UserNav
-          @user={{this.model}}
-          @showNotificationsTab={{this.showNotificationsTab}}
-          @showPrivateMessages={{this.showPrivateMessages}}
-          @canInviteToForum={{this.canInviteToForum}}
-          @showBadges={{this.showBadges}}
-          @currentParentRoute={{this.currentParentRoute}}
-          @showRead={{this.showRead}}
-          @showDrafts={{this.showDrafts}}
-          @showBookmarks={{this.showBookmarks}}
-        />
-
-        <div class="new-user-content-wrapper">
-          {{outlet}}
-        </div>
-      </div>
-    {{else}}
-      <div class="user-content-wrapper">
```

Comment:
I didn't remove the related CSS for this because the `/g/<:id>` routes are still using this.